app/gateways/db/memory/get_token.go

```diff
@@ -0,0 +1,22 @@
+package memory
+
+import (
+	"github.com/patriciapedrosaa/transfer-me/app/domain/entities"
+)
+
+func (m MemoryStorage) GetToken(id string) (entities.Token, error) {
+	token := m.storageAuthentication[id]
+	if token.ID == "" {
+		return entities.Token{}, errNotFound
+	}
```

Comment:
this won't fail or something, but i believe would be more concise if you do like this:
```suggestion
	_, ok := m.storageAuthentication[id]
	if !ok {
		return entities.Token{}, errNotFound
	}
```

as explained [here](https://blog.golang.org/maps#TOC_3.). What do you think?

Reply:
I agree, I believe it's easier to understand too